src/i18n/en-system-messages.ts

```diff
@@ -11,6 +11,8 @@ const systemMessagesMessageDescriptors = {
       ${SystemMessagesName.MotionHasFailedFinalizable} {{motionTag} has {failedTag} and may be finalized.}
       ${SystemMessagesName.MotionVotingPhase} {{votingTag} has started! Voting is secret 😀, and weighted by Reputation.}
```

Comment:
While you're at it, can you please change the voting system message emoji, so that Product is happy ? :)

It's the one listed in this issue: https://www.notion.so/colony/QA-008ecbf3acb84c6281ef8b619f5fcb34?p=058b0d7173554494b9c7365b29e1e235